modules/configure-acs-kube-security-platform.adoc

```diff
@@ -0,0 +1,52 @@
+[id="configure-acs-kube-security-platform"]
+= Configuring the {product-title} platform
+:context: configure-acs-kube-security-platform
+
+You can create a new integration in the {product-title} platform by using the email plugin notifier.
+
+.Procedure
+
+. Navigate to *Platform Configuration* -> *Integrations*.
+. Under the *Plugins* section, select *Email*.
+. Select the *New Integration* icon.
+. Enter a name for your email integration in the *Integration Name*.
+. For *Email Server*, enter the address of your email server.
```

Comment:
We should clarify more here. Address is the fully qualified domain name and the port <smtp.example.com:465> for example.